src/main/java/net/mcreator/element/types/Recipe.java

```diff
@@ -174,4 +180,27 @@ private MItemBlock[][] trim(MItemBlock[][] mtx, int rmin, int rmax, int cmin, in
 
 	}
 
+	@Override public Collection<? extends MappableElement> getUsedElementMappings() {
+		List<MappableElement> elements = new ArrayList<>();

```

Comment:
Depending on the recipe type, there are many fields that could be null, not just recipeSlots

Also not all references are always used, as this depends on recipe type.

Smelting recipe type will not use brewing stack for sure, even if it was defined.

Please go through all MEs and double check similar mistakes are not present in others.

Reply:
So not _all references should be collected_?

Reply:
Once again. 

> Smelting recipe type will not use brewing stack for sure, even if it was defined.

Why would we collect brewing recipe reference for smelting recipe? There is no type of Minecraft where smelting recipe needs brewing recipe input slot reference

Reply:
So this seems to go idle for some time then as I'd need to check all ME types for the 3rd time (2nd was 2-3 hours ago).

Reply:
✔️ Fixed